spec/requests/static_pages_spec.rb

```diff
@@ -31,16 +31,30 @@
 
   describe "#help" do
     it "should have the title Help" do
-      visit '/static_pages/help'
+      visit help_path
 
       expect(page).to have_title('Expenses | Help')
     end
 
     it "should have the content 'Help'" do
-      visit '/static_pages/help'
+      visit help_path
 
       expect(page).to have_content('Help')
     end
   end
 
+  describe "#contact" do
+    it "should have the title Help" do
+      visit contact_path
+
+      expect(page).to have_title('Expenses | Contact')
+    end
+
+    it "should have the content 'Contact'" do
+      visit contact_path
+
+      expect(page).to have_content('Contact')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.